### ISSUE TITLE
Outdated packages

### ISSUE THREAD

**Issue by yourivdlans** [nickcharlton/dokku-cookbook#19]
Thanks for the cookbook!

I was wondering if it would be possible to update all dependencies to their latest versions?
Some seem a bit outdated.

**Comment by yourivdlans** [comment]
For anyone interested I've looked at updating the packages myself.

I ran into a problem with `packagecloud`. Because of this https://github.com/computology/packagecloud-cookbook/issues/40 issue I was unable to install the additional packages required by dokku (e.g. herokuish). This is because I'm trying to install the cookbook on a Ubuntu 16.04 (Xenial) server. And the dokku [dokku packagecloud repository](https://packagecloud.io/dokku/dokku/) only has these packages for `ubuntu/trusty`.

Therefor I've looked at how packagecloud installs itself on the server and replicated this for the `dokku-cookbook`.

https://github.com/yourivdlans/dokku-cookbook/tree/usable-for-ubuntu-16.04

When the issue for packagecloud is resolved we could probably update the dependencies for this cookbook a bit easier.
